examples.md

```diff
@@ -225,12 +225,23 @@ We cache the elements of the Cabal store separately, as the entirety of `~/.caba
 
 ```yaml
 - uses: actions/cache@v3
+  if: startsWith(runner.os, 'Linux') || startsWith(runner.os, 'macOS')
   name: Cache ~/.stack
   with:
     path: ~/.stack
     key: ${{ runner.os }}-stack-global-${{ hashFiles('stack.yaml') }}-${{ hashFiles('package.yaml') }}
     restore-keys: |
       ${{ runner.os }}-stack-global-
+- uses: actions/cache@v3
```

Comment:
Can we make this separate for windows and unix systems, like done [here](https://github.com/actions/cache/pull/819/files#diff-cfb2db6f153a4484d9bbd408f3227a264dce68fadd747cecdc30bda012b4d8cdR195)?

Reply:
@Phantsure, can I double-check the intended URL behind the 'here'?

Reply:
@Phantsure, I'll revise the PR to reflect that.